vendor/lief/src/logging.hpp

```diff
@@ -91,15 +91,19 @@ class Logger {
 
   template <typename... Args>
   static void err(const char *fmt, const Args &... args) {
+    const char *prefix = "Error: ";
+    const std::string prefixed_fmt = std::string(prefix) + std::string(fmt);
     if /* constexpr */ (lief_logging_support) {
-      Logger::instance().sink_->error(fmt, args...);
+      Logger::instance().sink_->error(prefixed_fmt.c_str(), args...);
     }
   }
 
   template <typename... Args>
   static void warn(const char *fmt, const Args &... args) {
+    const char *prefix = "Warning: ";
+    const std::string prefixed_fmt = std::string(prefix) + std::string(fmt);
     if /* constexpr */ (lief_logging_support) {
-      Logger::instance().sink_->warn(fmt, args...);
+      Logger::instance().sink_->warn(prefixed_fmt, args...);
```

Comment:
Hey @RaisinTen 👋🏼 

I really like to have your view on two things:
- the string concatenation: did I use the right tool for that?
- do you have an idea about how could I generate LIEF errors to see them appear somewhere?

Reply:
> the string concatenation: did I use the right tool for that?

Yes, it looks okay to me :+1:

> do you have an idea about how could I generate LIEF errors to see them appear somewhere?

You could try to temporarily patch the source code locally to add a `LIEF_ERR()` call somewhere and build it from source and run it?

Reply:
<img width="130" alt="Screenshot 2023-02-25 at 18 09 11" src="https://user-images.githubusercontent.com/22824417/221370230-9ae9be4f-0342-4917-864f-5fb836243df4.png">
<img width="155" alt="Screenshot 2023-02-25 at 18 09 16" src="https://user-images.githubusercontent.com/22824417/221370231-25b9cfa7-20de-42dc-847b-9e3380f6151c.png">

Seems good ^^